src/main/scala/beam/router/r5/R5RoutingWorker.scala

```diff
@@ -1050,14 +1057,24 @@ class R5RoutingWorker(workerParams: WorkerParameters) extends Actor with ActorLo
       )
 
     val travelSpeed = network.getLinks.get(Id.createLinkId(linkId)).getLength / travelTime
-    if (travelSpeed < beamServices.beamConfig.beam.physsim.quick_fix_minCarSpeedInMetersPerSecond) {
+    travelTime = if (travelSpeed < beamServices.beamConfig.beam.physsim.quick_fix_minCarSpeedInMetersPerSecond) {
       network.getLinks
         .get(Id.createLinkId(linkId))
         .getLength / beamServices.beamConfig.beam.physsim.quick_fix_minCarSpeedInMetersPerSecond
     } else {
       travelTime
     }
 
+    if (pureTravelTimeWithoutCosts) {
+      travelTime
+    } else {
+      val tollPriceFile = beamServices.beamConfig.beam.agentsim.toll.file
+      val tollPrices = BridgeTollDefaults.readTollPrices(tollPriceFile) // TODO: load tolls file first time only, as soon as logic works
+      val valueOfTravelTimesSavings = 10.0 // 10$/h
+
+      travelTime + tollPrices.getOrElse(linkId, 0.0).asInstanceOf[Double] / valueOfTravelTimesSavings * 3600
```

Comment:
This happens to work, but I'd feel better to guarantee ordering by parentheses

    travelTime + ((tollPrices.getOrElse(linkId, 0.0).asInstanceOf[Double] / valueOfTravelTimesSavings) * 3600)